sql/moz-fx-data-shared-prod/firefox_desktop_derived/newtab_content_items_daily_v1/query.sql

```diff
@@ -0,0 +1,89 @@
+WITH events_unnested AS (
+  SELECT
+    DATE(submission_timestamp) AS submission_date,
+    -- mozfun.norm.browser_version_info(client_info.app_display_version).major_version AS app_version,
+    140 AS app_version,  -- Placeholder for app_version, adjust as needed
+    normalized_channel AS channel,
+    -- metrics.string.newtab_locale AS locale,
+    "MISSING" AS locale,  -- Placeholder for locale, adjust as needed
```

Comment:
Since we may not be adding locale to the `newtab_content` ping, we can perhaps remove it from the code